tests/e2e/operator/operator_test.go

```diff
@@ -99,33 +107,58 @@ var _ = Describe("Operator", Ordered, func() {
 		})
 
 		It("deploys all the CRDs", func() {
```

Comment:
It looks like we can get the context from ginkgo. This allows it to terminate the test when requested by the user:

```suggestion
		It("deploys all the CRDs", func(ctx SpecContext) {
```